test/index.ts

```diff
@@ -169,6 +169,13 @@ const runTest = async (data: typeof map.dotnet, envStr = "") => {
       console.error("Azure CLI Not Found.")
       process.exit(1);
     }
+    // Powershell module tests Az module
+    if (imageName.indexOf("powershell") !== -1) {
+      if (parseInt(shell.exec(`docker run ${name} pwsh -Command "(Get-InstalledModule).count"`).stdout) === 0){
```

Comment:
You may want to be more specific here: `Get-InstalledModule -Name 'Az'`, otherwise this code is not really checking for Az.

Reply:
Thanks. Will do!